release-please-config.json

```diff
@@ -0,0 +1,79 @@
+{
+  "bootstrap-sha": "77ecc2590ea9899db4f47a92ca1bbb50c18c6777",
```

Comment:
I don't think we want this, it doesn't point to a commit in this repo. We could probably try the commit on `main` before this PR is merged, docs: https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#bootstrapping or just try without this value and see what happens :shrug: